### ISSUE TITLE
Submissions on the Phoenix App

### ISSUE THREAD

**Issue by r-bartlett-gsa** [GSA/Challenge_platform#171]
Note: This work needs to be deployed to [challenge portal eval dev environment](https://challenge-portal-eval-dev.app.cloud.gov/) ([challenge_gov repo > eval main branch](https://github.com/GSA/Challenge_gov/tree/eval-main))

### User story

As a challenge manager, in order to be able to review and manage submissions and evaluations, I would like to access them from any Challenge.gov app and be sent to the appropriate section.

**Acceptance criteria:**

- [x] A challenge manager is dynamically based on environment routed to the Rails app > Submissions list when they click on view submissions on the Phoenix app
  - [x] https://challenge-dev.app.cloud.gov/phases/**13**/submissions (13 in this link means challenge phase ID)
- [x] A challenge manager is dynamically based on environment routed to the Rails app > Submissions details page when they try to access submissions information on the Phoenix app
  - [x] https://challenge-dev.app.cloud.gov/submissions/**16** (16 in this link means submission ID)
- [x] Submissions access remains the unchanged for Admin and Super Admin users

### Definition of Done

**Doing (dev team)**
- [x] Code complete 
- [x] Code is organized appropriately
- [x] Any known trade offs are documented in the associated GH issue
- [x] Code is documented, modules, shared functions, etc.
- [x] Automated testing has been added or updated in response to changes in this PR
- [x] The feature is smoke tested to confirm it meets requirements
- [x] Database changes have been peer reviewed for index changes and performance bottlenecks
- [x] PR that changes or adds UI
  - [x] include a screenshot of the WAVE report for the altered pages
  - [x] Confirm changes were validated for mobile responsiveness
- [x] PR approved / Peer reviewed 
- [x] Security scans passed
- [x] Automate accessibility tests passed
- [x] Build process and deployment is automated and repeatable
- [x] Feature toggles if appropriate
- [x] Deploy to staging
- [x] Move card to testing column in the board
---
**Staging**
- [ ] Accessibility tested (Marni)
  - [ ]  Keyboard navigation
  - [ ]  Focus confirmed
  - [ ] Color contrast compliance
  - [ ] Screen reader testing 
- [ ] Usability testing: mobile and desktop (Tracy or Marni)
- [ ] Cross browser testing - UI rendering is performant on below listed devices/browsers (Tracy or Marni)
  - [ ] Windows/Chrome
  - [ ] Windows/Edge
  - [ ] Mac/Chrome
  - [ ] Mac/Safari
  - [ ] iOS/Safari
- [ ] AC review (Renata)
- [ ] Deploy to production (production-like environment for eval capability) (dev team)
- [ ] Move to production column in the board
---
**Production**
- [ ] User and security documentation has been reviewed for necessary updates (Renata / Tracy / Dev team) 
- [ ] PO / PM approved (Jarah or Renata)
- [ ] AC is met and it works as expected (Jarah or Renata)
- [ ] Move to done column in the board (Jarah or Renata)

**Comment by TCKapGrp** [comment]
@jairoanaya, revised the AC to reflect dynamic updating of the url.

**Comment by jairoanaya** [comment]
Yes, this feature update includes a dynamic URL depending on the environment.

**Comment by afeijoo** [comment]
Archiving repo; no longer applicable.
